doc/papers/joss/paper.md

```diff
@@ -103,11 +103,13 @@ at run time). This easily allows $hp$-refinement studies (where $h$ commonly den
 elements with different topology. This sketch shows the independence of QFunctions
 (in this case representing a Laplacian) element resolution, topology, or basis degree.\label{fig:schematic}](img/QFunctionSketch.pdf)
 
-LibCEED is a C99 library with Fortran77, Python, and Julia interfaces.
+LibCEED is a C99 library with Fortran77, Python, Julia, and Rust interfaces.
 
 The Python interface was developed using the C Foreign Function Interface (CFFI) for Python. CFFI allows to reuse most of the C declarations and requires only a minimal adaptation of some of them. The C and Python APIs are mapped in a nearly 1:1 correspondence. For instance, a ``CeedVector`` object is exposed as ``libceed.Vector`` in Python, and may reference memory that is also accessed via Python arrays from the NumPy [@NumPy] or Numba [@Numba] packages, for handling host or device memory (when interested in GPU computations with CUDA). Flexible pointer handling in libCEED makes it easy to provide zero-copy host and (GPU) device support for any desired Python array container. The interested reader can find more details on libCEED's Python interface in [@libceed-paper-proc-scipy-2020].
 
-The Julia interface, referred to as ``LibCEED.jl``, provides both a low-level interface, which is generated automatically from ``libCEED``'s C header files, and a high-level interface. The high-level interface takes advantage of Julia features such as garbage collection to simplify memory management. A key feature of ``LibCEED.jl`` is the straightforward creation of user Q-functions that work on both the CPU and GPU, making use of Julia's metaprogramming and just-in-time compilation capabilities [@Julia-2017].
+The Julia interface, referred to as ``LibCEED.jl``, provides both a low-level interface, which is generated automatically from ``libCEED``'s C header files, and a high-level interface. The high-level interface takes advantage of Julia features such as garbage collection to simplify memory management. A key feature of ``LibCEED.jl`` is the straightforward creation of user QFunctions that work on both the CPU and GPU, making use of Julia's metaprogramming and just-in-time compilation capabilities [@Julia-2017].
+
+The Rust interface provides a high-level interface which wraps the automatically generated bindings from the ``libCEED`` C header files. The Rust interface takes advantage of ownership and borrow checking in Rust to provide improved memory safety and uses closures to simplify management of user provided environment variables in the context of the user QFunctions.
 
```

Comment:
Thanks @jeremylt ! Is there a Rust reference to cite here?

Reply:
Not that I'm aware of? We aren't using anything that isn't a core language feature.

Reply:
I meant even just a dev website, if there is no paper or no official way to cite them explicitly mentioned on their website (like we did for other languages or packages we mentioned in this paper). We should not assume every reader is familiar with what Rust is.

Reply:
Do we want to also include citations on what Python, C, and Fortran77 are? Those are the other languages mentioned but not cited directly.

Reply:
Ahah that is true. I just thought it might be a bit unheard of for some readers since it's relative new. I am always in favor of more references than less :)

Reply:
I unified our approach to citing languages. What do you think?

Reply:
Thanks Jeremy. I thought you were being sarcastic when you mentioned to cite all of the languages 😂 ah, the joy of online conversations with no tonal and/or visual aid. 

To me it's ok. I guess we can hear @jedbrown 's opinion, too, when he gets the chance.

Reply:
I was sort of on the fence, but you like references and I like uniformity, so it seemed a good way to satisfy both.

Reply:
Seems fine to me.